### DISCUSSION TITLE
Hacia una Inteligencia Artificial Responsable

### DISCUSSION THREAD

**Discussion by David-Lazaro-Fernandez** [gittogethers/octogatosconf22#71] [Talks in Spanish]
***Ponente: Luis Beltran***
> "Cuando hablamos de IA, generalmente nos referimos a un modelo de machine learning que se usa dentro de un sistema para automatizar algo. Por ejemplo, un automóvil autónomo puede tomar imágenes de sensores. Un modelo de ML puede usar estas imágenes para hacer predicciones (ejemplo: el objeto que tenemos delante es un árbol). El automóvil utiliza estas predicciones para tomar decisiones (ejemplo: girar a la izquierda para evitar el árbol). A todo este sistema nos referimos como Inteligencia Artificial.
>
>  
> 
>
>
>  
>
> Esto es sólo un ejemplo. La IA se puede utilizar para cualquier cosa, desde la suscripción de seguros hasta la detección del cáncer. La característica definitoria es que no hay participación humana limitada en las decisiones que toma el sistema. Esto puede dar lugar a muchos problemas potenciales y las empresas deben definir un enfoque claro para el uso de la IA. La IA Responsable es un framework de gobernanza destinado a hacer exactamente eso.
>  
>  
>  
>  La IA Responsable puede incluir detalles sobre qué datos se pueden recopilar y usar, cómo se deben evaluar los modelos y cómo implementar y monitorear los modelos de la mejor manera. También puede definir quién es responsable de los resultados negativos de la IA. Puede definir enfoques específicos y otros más abiertos a la interpretación. Todos buscan lograr lo mismo: crear sistemas de IA que sean interpretables, justos, seguros y respetuosos con la privacidad del usuario.
>  
>  
>  
>  El objetivo de esta sesión es hablar del uso responsable de la Inteligencia Artificial en la generación de modelos de machine learning justos, equitativos y explicables."

**Comment by jsiesquen** [comment]
Cada solución de IA debería de mostrar datos de como está diseñado su solución y en que principios se basa?
Donde puedo ver que esta normalizado internacionalmente.

**Reply by mcnanton** [reply]
Quizas te interese este artículo que propone una manera de documentar las características, limitaciones y ámbito de aplicación de un modelo de IA https://arxiv.org/pdf/1810.03993.pdf
